src/include/sof/audio/pipeline.h

```diff
@@ -42,6 +42,13 @@
 #include <sof/schedule.h>
 #include <uapi/ipc/topology.h>
 
+/*
+ * This flag disables firmware-side xrun recovery.
+ * It should remain enabled in the situation when the
+ * recovery is delegated to the outside of firmware.
```

Comment:
this wording is award "this flag disables... should remain enabled". "outside of firmware" isn't very nice either